src/pages/MyLists/container.js

```diff
@@ -4,8 +4,10 @@ import PropTypes from 'prop-types';
 
 import MyLists from './component';
 import {
-  fetchMyLists as fetchMyListsAction,
   createMyList as createMyListAction
+} from '../../../state/my_list/actions';
```

Comment:
I think you also should change mock in the test

Reply:
Removed useless props in tests as we discussed.